appserver/jdbc/jdbc-ra/jdbc-core/src/main/java/com/sun/gjc/spi/JdbcObjectsFactory.java

```diff
@@ -52,13 +52,14 @@ public abstract class JdbcObjectsFactory implements Serializable {
      * @return JdbcObjectsFactory
      */
     public static JdbcObjectsFactory getInstance() {
+        String className = "com.sun.gjc.spi.jdbc40.Jdbc40ObjectsFactory";
         try {
             return(JdbcObjectsFactory)
-                Class.forName("com.sun.gjc.spi.jdbc40.Jdbc40ObjectsFactory")
+            Class.forName(className)
                      .getDeclaredConstructor()
                      .newInstance();
         } catch (Exception e) {
-            _logger.log(WARNING, "jdbc.jdbc_factory_class_load_exception", e);
+            _logger.log(WARNING, "jdbc.jdbc_factory_class_load_exception", className);
```

Comment:
As I am fighting with classloaders now, could you log the exception too? Sometimes it has an important cause.

Reply:
Yes no problem, I understand.
Is it ok to make the change and squash the commit?